dijet/selection/default.py

```diff
@@ -105,15 +100,15 @@ def default(
     events = self[process_ids](events, **kwargs)
 
     # build categories
-    events = self[category_ids](events, results=results, **kwargs)
+    # events = self[category_ids](events, results=results, **kwargs)
```

Comment:
We add the categories in `ProduceColumns` now, right? In that case, we could remove it here, although I suspect we might still need to produce the `category_ids` column during `SelectEvents`, since otherwise `PlotCutflow` and related tasks might break. Could also be a dummy column containing the inclusive category ID.

The above does not need to be implemented as part of this PR, but maybe we can add a comment here saying why it is commented out?